session/manager.go

```diff
@@ -131,6 +131,7 @@ func (manager *Manager) Create(consumerID identity.Identity, issuerID identity.I
 	sessionInstance.ConsumerID = consumerID
 	sessionInstance.done = make(chan struct{})
 	sessionInstance.Config = config
+	sessionInstance.ProposalID = proposalID
```

Comment:
I don't think we this field in session, it's a serial number for proposal but not unique index itself

Reply:
It's not supposed to be unique index, we have session id for that. This field allows finding which sessions belong to service being stopped.

Reply:
Oh, now I got what you meant - will fix this.